core/src/main/scala/org/apache/spark/h2o/ui/H2ORuntimeInfoUIThread.scala

```diff
@@ -26,9 +26,13 @@ import water.H2O
   */
 class H2ORuntimeInfoUIThread(sc: SparkContext, conf : H2OConf) extends Thread{
   override def run(): Unit = {
-    while(!Thread.interrupted()){
+    while(!Thread.currentThread().isInterrupted){
```

Comment:
I do not fully understand the motivation here. Why you need to re-interrupt current thread and reread status again? A simple flag should be enough, no?